components/server/src/prebuilds/github-app.ts

```diff
@@ -570,7 +570,7 @@ export class GithubApp {
 
         const newBody = body + `\n\n${button}\n\n`;
         const updatePrPromise = ctx.octokit.pulls.update({ ...ctx.repo(), pull_number: pr.number, body: newBody });
-        updatePrPromise.catch((err) => log.error(err, "Error while updating PR body", { contextURL }));
+        updatePrPromise.catch((err) => log.error("Error while updating PR body", err, { contextURL }));
```

Comment:
updating the type signatures of our log messages revealed that we were passing the arguments in the wrong order

Reply:
maybe we should enable linting rule to disable usage of console on server

Reply:
yes, would make sense in a different PR